File: src/main/scala/analysis/advertising/bean/AdProvinceTop3.scala
package analysis.advertising.bean

/**
 * 各省top3热门广告
 *
 * @param date
 * @param province
 * @param adid
 * @param clickCount
 */
case class AdProvinceTop3(date: String,
                          province: String,
                          adid: Long,
                          clickCount: Long
                         )

File: src/main/java/com/jxust/entity/AdProvinceTop3.java
package com.jxust.entity;

import java.io.Serializable;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;


/**
* Create by Jayden in 2020/3/7
*/

@Data
@AllArgsConstructor
@NoArgsConstructor
public class AdProvinceTop3 implements Serializable {
    private String date;

    private String province;

    private Integer adid;

    private Integer clickcount;

    private static final long serialVersionUID = 1L;
}